tests/tape/tapes/test_qnode.py

```diff
@@ -324,7 +324,57 @@ def func(x, y):
         assert qn.qtape.operations == [op1, op2, op3]
         assert qn.qtape.measurements == [m1, m2]
 
+    def test_drawing(self):

```

Comment:
Maybe add a test for the ASCII drawing as well?

Reply:
done!